development/tsdb-blocks-storage-s3/config/prometheus.yaml

```diff
@@ -56,3 +56,5 @@ scrape_configs:
 remote_write:
   - url: http://distributor-1:8000/api/v1/push
     send_exemplars: true
+    headers:
```

Comment:
For testing locally. I wasn't sure if this should be set long term, I can remove them if desired.

Reply:
Will do!